### DISCUSSION TITLE
Add a cheat for faster ride research -- or -- additional research funding levels

### DISCUSSION THREAD

**Discussion by ocalhoun6** [OpenRCT2/OpenRCT2#19024] [Feature requests]
Often, when casually playing scenarios, I find myself sitting around and waiting for the next ride to be researched.

I know, of course, that there are already cheats for 'research all rides' and that it's also possible to manually edit the invention list and move things to the 'invented' section of it. *But* those take all the fun out of actually doing the R&D to get new rides. Heck, I could just as easily add rides that were never meant to be in that scenario in the first place. Yeah, I *could* just use those methods ... but it takes some of the fun out of playing the scenario. I'd really like to keep the same research game mechanics ... but faster.

What I'd like to see is something like a cheat for 'double research speed' or 'triple research speed', which would act as a multiplier against the research level you're already paying for.

------

Alternately, it would be nice to have additional higher funding levels for the research, added as a non-cheat option. For example, adding a $800/mo option that researches rides twice as fast as the current $400/mo 'maximum' funding level.

The new level could be named 'overtime' perhaps? Or maybe the current 'maximum' could be renamed to 'very high' while the new $800/mo level takes on the name 'maximum'.

(I think it's reasonable enough from a realism standpoint. With double the budget, you could hire two independent R&D teams and get twice as much research done.)

**Comment by 733737** [comment]
I still believe OpenRCT2 would benefit from a "Vanilla Toggle" system, while I have suggested it in the past, it hasn't been thoroughly investigated. This could add more research modes, different stat calculation, or other gameplay balance changes such as 25.50 maximum ride tickets. Currently there isn't much room for game balance changes, so I don't see your suggestion being implemented without a game balance toggle.

**Reply by ocalhoun6** [reply]
Putting it in the cheat menu would alleviate game balance concerns ... since there are already things in the cheat menu that affect the game balance *far* more.
